docs/gallery/gallery/scales/custom_scales.jl

```diff
@@ -9,9 +9,23 @@ using AlgebraOfGraphics, CairoMakie
 using Colors
 set_aog_theme!() #src
 
-# Sometimes, there is no default palettes for a specific attribute. In that
-# case, the user can pass their own. TODO: allow legend to use custom attribute
-# of plot, such as the arrowhead or the arrowcolor and pass correct legend symbol.
+
+# A pallette maps particular values to particular attribute specifications (e.g.
+# 1 maps to green, 2 maps to red). Sometimes, there is no default pallette for a
+# specific attribute, and you will need to specify it manually, but there are
+# sensible default pallettes for many attributes. In either case, you can always
+# manually specify the pallette used for a particular attribute.
+
+# TODO: allow legend to use custom attribute of plot, such as the arrowhead or
+# the arrowcolor and pass correct legend symbol.
+
+# !!! note
+
```

Comment:
```suggestion
```

Just to be sure the conversion to markdown does not screw up the note formatting.